JM/CF/Scripts/3_Game/CommunityFramework/Expression/DayZGame.c

```diff
@@ -0,0 +1,22 @@
+modded class DayZGame
+{
+#ifdef CF_EXPRESSION_TEST
```

Comment:
The `#ifdef` should wrap the whole file, as we do not need to add a modded DayZGame if the tests are not run.